tests/test_mdib.py

```diff
@@ -2,6 +2,11 @@
 import os
 from sdc11073 import mdib
 from sdc11073 import pmtypes
+from lxml import etree as etree_
+
+from sdc11073.mdib import ClientMdibContainer

```

Comment:
unused import

also a module import would be preferred in case of it being used

Reply:
removed unused import